docs/DeveloperGuide.adoc

```diff
@@ -12,255 +12,359 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S2-CS2103T-W17-2/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team Notably`      Since: `Feb 2020`      Licence: `MIT`
 
 == Setting up
 
 Refer to the guide <<SettingUp#, here>>.
 
 == Design
 
+[TIP]
+The `.puml` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder.
+Refer to the <<UsingPlantUml#, Using PlantUML guide>> to learn how to create and edit diagrams.
+
 [[Design-Architecture]]
 === Architecture
 
 .Architecture Diagram
```

Comment:
Would be good if the architecture diagram was used straight lines instead of curved ones, so that the diagram is less confusing.

Reply:
The architecture diagram seems a bit confusing.  Could draw clearer where the commons is coming from! Otherwise, the architecture diagram seems to have been refactored greatly, keep up the hard work!